src/components/layout/BottomModal.tsx

```diff
@@ -229,7 +229,7 @@ const BottomModal = ({ Content, isOpen, onClose, maximisedContent, customMinHeig
               <Navigation style={modalNavigationAnimatedStyle}>
                 <Button onPress={handleClose} iconProps={{ name: 'close-outline' }} round />
               </Navigation>
-              <KeyboardAvoidingView style={{ flex: 1 }} behavior="position">
+              <KeyboardAvoidingView style={{ flex: 1 }} behavior="height">
```

Comment:
Android is broken

<img width="487" alt="image" src="https://github.com/alephium/mobile-wallet/assets/1579899/30044481-d15b-4ad3-82bf-4f960b1ae4ec">